src/onepasswordconnectsdk/client.py

```diff
@@ -3,6 +3,8 @@
 from httpx import HTTPError
 import json
 import os
+from typing import Optional, Union, overload
+from typing_extensions import Literal
```

Comment:
`typing.Literal` was added in Python 3.8, which is why we need the backported version here.

If this project were to drop support for 3.7, the import could be updated.